test/WebJobs.Script.Tests/SamplesEndToEndTests.cs

```diff
@@ -38,6 +40,42 @@ public SamplesEndToEndTests(TestFixture fixture)
         }
 
         [Fact]
+        public async Task EventHubTrigger_Batch()
+        {
+            TestHelpers.ClearFunctionLogs("EventHubTrigger-Batch");
+
+            // write 3 events
+            List<EventData> events = new List<EventData>();
+            string[] ids = new string[3];
+            for (int i = 0; i < 3; i++)
+            {
+                ids[i] = Guid.NewGuid().ToString();
+                JObject jo = new JObject
+                {
+                    { "value", ids[i] }
+                };
+                events.Add(new EventData(Encoding.UTF8.GetBytes(jo.ToString(Formatting.None))));
+            }
+
+            string connectionString = Environment.GetEnvironmentVariable("AzureWebJobsEventHubReceiver");
+            string eventHubName = "testhub";
```

Comment:
This test runs great in isolation, but since there is another sample function listening to the same hub, they don't run together. I'll have to either create a new hub, or perhaps use direct invocation. TBD